api/api.proto

```diff
@@ -3097,7 +3191,7 @@ message SdkVersion {
     // SDK version major value of this specification
     Major = 0;
     // SDK version minor value of this specification
-    Minor = 43;
+    Minor = 44;
```

Comment:
fyi, we will be merging storage policies which will be 0.44.0, so this will need a rebase.